README.md

```diff
@@ -24,32 +57,119 @@ running command...
 
 Now you should be able to run functions commands, e.g. `sf generate project`, `sf login functions`
 
+# Contributing
+
+1. Please read our [Code of Conduct](CODE_OF_CONDUCT.md)
+2. Create a new issue before starting your project so that we can keep track of
+   what you are trying to add/fix. That way, we can also offer suggestions or
+   let you know if there is already an effort in progress.
+3. Fork this repository.
+4. [Build the plugin locally](#build)
+5. Create a _topic_ branch in your fork. Note, this step is recommended but technically not required if contributing using a fork.
+6. Edit the code in your fork.
+7. Create commit messages that adhere to the [conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/).
+8. Write appropriate tests for your changes. Try to achieve at least 95% code coverage on any new code. No pull request will be accepted without unit tests.
+9. Sign CLA (see [CLA](#cla) below).
+10. Send us a pull request when you are done. We'll review your code, suggest any needed changes, and merge it in.
+
+## CLA
+
+External contributors will be required to sign a Contributor's License
+Agreement. You can do so by going to https://cla.salesforce.com/sign-cla.
+
+# Build
+
+To build the plugin locally, make sure to have yarn installed and run the following commands:
+
+```bash
+# Clone the repository
+git clone git@github.com:salesforcecli/plugin-functions
+
+# Install the dependencies and compile
+yarn install
+yarn build
+```
+
+To use your plugin, run using the local `./bin/run` or `./bin/run.cmd` file.
+
+```bash
+# Run using local run file.
+./bin/run npm
```

Comment:
If I run this command as shown, it doesn't work (npm not found). I do have npm on my system, and it works fine if I just run it directly. If there is some additional setup/specifics needed for npm, note them; or perhaps use a different example.